poetry.lock

```diff
@@ -1087,6 +1087,37 @@ nodeenv = ">=1.6.0"
 all = ["twine (>=3.4.1)"]
 dev = ["twine (>=3.4.1)"]
 
+[[package]]
+name = "pysha3"
```

Comment:
Why use [`pysha3`](https://pypi.org/project/pysha3/)? It is [no longer maintained](https://github.com/tiran/pysha3).

Also, why isn't the `pyproject.toml` file updated, but only the `poetry.lock` file is?

Reply:
I used `pysha3` library because the ETH checksum address validation is based on `sha3.keccak_256` method.

I left updating `pyproject.toml` for the package owners so as not to mess with the organization of dependency groups. Feel free to add `pysha3` to `pyproject.toml` as you wish.